src/pages/index.js

```diff
@@ -98,8 +216,14 @@ addButton.addEventListener("click", () => {
 });
 
 profileEditButton.addEventListener("click", () => {
-  const userData = profileInfo.getUserInfo();
-  profileTitleInput.value = userData.title;
-  profileDescriptionInput.value = userData.description;
+  const userData = userInfo.getUserInfo();
+  profileTitleInput.value = userData.name;
+  profileDescriptionInput.value = userData.about;
   profileModal.open();
+  profileFormValidator.disableSubmitButton();
+});
+
+profileAvatarModalButton.addEventListener("click", () => {
+  profileAvatarModal.open();
+  profilePictureModalValidator.disableSubmitButton();
```

Comment:
Please, use `toggleSubmitButton` instead of `disableSubmitButton` on opening because  `toggleSubmitButton`  will disable the button only if the form has been `reset`, so it's more flexible




```js

addButton.addEventListener("click", () => {
  addCardFormValidator.toggleSubmitButton();
  addCardModal.open();
});



profileAvatarModalButton.addEventListener("click", () => {
  profileAvatarModal.open();
  profilePictureModalValidator.toggleSubmitButton();
});

```